src/utils/wait-for-deployment-url.ts

```diff
@@ -1,8 +1,11 @@
 import { getOctokit } from "@actions/github";
 import { GitHub } from "@actions/github/lib/utils";
+import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
```

Comment:
Doesn't look like this or line 6 is used in the new changes, might be unused imports? Or did you mean to use `log.getLogger` here rather than calling `console.error`?